### ISSUE TITLE
[PROCESSED IE] Implement changes to flyout and filter panel for Processed IEs

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#3370]
**What we're after:** As part of our work to allow users to filter and view transactions that are the latest version, we need to implement several changes to the front end filter and flyout panels on the IE datatable. 

As part of this work, we need to make changes to both the IE datatable Raw AND Processed panels. The work in this ticket contains changes to the Processed filter panel. [Raw IE work happening here.](https://github.com/fecgov/fec-cms/issues/3011)

**Completion criteria:**
Filter panel
- [ ] ~Modify the Regularly Scheduled Report Form to gray out when the 24- and 48-hour reports toggle is active~ This has been moved to a new issue for filter & tags interaction work: https://github.com/fecgov/fec-cms/issues/3448
- [x] Link below version filter is a pop-up similar to other methodology pop-ups https://github.com/fecgov/fec-cms/issues/3360
- [x] Add version filter according to mock-up below
- [x] Update methodology text according to this issue for methodology: https://github.com/fecgov/fec-cms/issues/3360
- [x] Current version filter should be checked by default on page load

Flyout panel
- [x] Change "Open original image" to "Open image" 
- [x] Update the flyout panel according to the mockup below to include version type

#### Flyout panel
<img width="863" alt="Screen Shot 2019-11-18 at 8 45 40 AM" src="https://user-images.githubusercontent.com/31663028/69057424-d7236700-09df-11ea-998a-a5647e487b23.png">

#### Processed filter panel
<img width="278" alt="Screen Shot 2019-11-19 at 3 44 33 PM" src="https://user-images.githubusercontent.com/31663028/69184785-8432d800-0ae3-11ea-923c-8fb7c72e0300.png">

### Methodology pop-up
TBD

**Comment by patphongs** [comment]
The expanding filters are not working properly when I apply them to the raw IE filters. They break the expanding filters on the processed data. I think to fix this, we would need to overhaul how the expanding filter show/hides are working.

**Comment by patphongs** [comment]
@AmyKort @PaulClark2 @JonellaCulmer The accordions for the raw data filters will be a blocker on the raw vs processed data toggle. Right now, the accordions are breaking on the processed data filters when I add the accordion to the raw data filters. This is a major problem as I’ll need to debug and refactor the accordions (which will probably effect other places on the site, so need to make sure those are tested too).

**Bottom line:** I’m going to try and make the accordions work, although I think it’s a big lift. However, if I cannot make it work within this sprint, I suggest that we move forward with just implementing the current version filter and methodology first on both the processed and raw data filters. Then split out the work to add the other filters later in another sprint.

**Comment by patphongs** [comment]
@AmyKort @PaulClark2 @JonellaCulmer I was able to get the accordions to work correctly between raw and processed data toggles. However, I've found yet another bug where the filter counts on the accordion are double counting filters within the accordion itself. See this new issue I logged for this: https://github.com/fecgov/fec-cms/issues/3465

Since this is a new bug I just discovered, I don't think I can debug and fix it fully this sprint. We may want to prioritize it for next sprint. I've added a feature flag for the additional raw IE filters, so my work won't be lost, but the added filters won't show up until we fix the count bug and turn it on.

**Comment by patphongs** [comment]
This work has been completed and merged in
